### DISCUSSION TITLE
How i can display INFO.md on my sidebar?

### DISCUSSION THREAD

**Discussion by jonathanbergson** [vuepress/core#614] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/6494405/148453114-9681cb28-e939-4a13-9af8-cd455c06eb84.png)

**Comment by Mister-Hope** [comment, marked as answer]
You can only display it when it's inside your vuepress project folder.

If you do want to display something outside of it, maybe a small script copying them to project folder is your consideration.
